be/src/olap/schema_change.cpp

```diff
@@ -266,6 +260,92 @@ bool RowBlockChanger::change_row_block(
         int32_t ref_column = _schema_mapping[i].ref_column;
 
         if (_schema_mapping[i].ref_column >= 0) {
+            if (!_schema_mapping[i].materialized_function.empty()) {
```

Comment:
Would better abstract the materialized_view logic to a method. Add add more comment.